static/css/main.css

```diff
@@ -288,6 +288,16 @@ a.button.inverse
 {
   min-height: 4em;
   background-color: #c70d2c;
+  position: fixed;
+  z-index: 2;
+  width: 100%;
+  top: 0;
+  transition: all 0.2s ease-in-out;
+ }
+
+#navbar.hide-navbar
+{
+    top: -4em;
```

Comment:
NIT: The `#navbar` height is a min-height because `#navbar-menu` expands vertically on small screens. As a result, we cannot garantee that `top: -4em;` will hide the navbar (because min-height is not height).

Speaking of which, this seems to function strangely on mobile. If you open the menu and then scroll then the menu will remain and the navbar will hide. I guess it should stay in place on smaller screens (what do you think)?  And it seems this change somehow prevents us from scrolling the expanded list of languages in the menu 😕.

Reply:
It doesn't seem to expand in any of the mobile options in the dev tools, but yeah, if I minimize the screen extremely, I see that.
I agree that we can leave the navbar fixed in the smaller screens (is that what you mean?), so - max width 991px? smaller? That will solve also the -4em problem, won't it?

Reply:
> It doesn't seem to expand in any of the mobile options in the dev tools, but yeah, if I minimize the screen extremely, I see that.

1. Shrink window down to mobile phone size
2. Expand navbar menu
3. Expand locale menu in navbar

> I agree that we can leave the navbar fixed in the smaller screens (is that what you mean?), so - max width 991px? smaller? That will solve also the -4em problem, won't it?

- Yes, leave the navbar visible (fixed) on smaller screens
- Yes, max-width 991
- No, that will not solve the -4em problem that I mentioned above
  - (Solving the -4em problem is not necessary to resolve this issue)
  - (To solve the -4em problem, one must put everything in the navbar except the navbar menu (currently only the navbar logo and the menu toggle button/link) into a transparent container that has a fixed height and apply the same fixed height to the navbar when the window is more than 992px wide. After that, the min-height will no longer be necessary. Does that make sense?)